net-print/brother-mfc9330cdw-bin/brother-mfc9330cdw-bin-1.1.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit rpm
+
+DESCRIPTION="Brother printer driver for MFC-9330CDW"
+
+HOMEPAGE="http://support.brother.com"
+
+SRC_URI="http://download.brother.com/welcome/dlf100399/mfc9330cdwlpr-1.1.2-1.i386.rpm
+	http://download.brother.com/welcome/dlf100401/mfc9330cdwcupswrapper-1.1.2-1.i386.rpm"
+
+LICENSE="brother-eula"
+
+SLOT="0"
+
+KEYWORDS="amd64"
+
+IUSE=""
+
+RESTRICT="mirror strip"
+
+DEPEND="net-print/cups"
+RDEPEND="${DEPEND}"
+
+S=${WORKDIR}
+
+src_unpack() {
+	rpm_unpack ${A}
+}
+
+src_install() {
+	has_multilib_profile && ABI=x86
+
+	dosbin "${WORKDIR}/usr/bin/brprintconf_mfc9330cdw"
+
+	cp -r usr "${D}" || die
+	cp -r opt "${D}" || die
+
+
+	mkdir -p ${D}/usr/libexec/cups/filter || die
+	( cd ${D}/usr/libexec/cups/filter/ && ln -s ../../../../opt/brother/Printers/mfc9330cdw/lpd/filtermfc9330cdw brlpdwrappermfc9330cdw ) || die
```

Comment:
The symlink is create here. Can you try to replace the line with the following:
( cd ${D}/usr/libexec/cups/filter/ && ln -s ../../../../opt/brother/Printers/mfc9330cdw/lpd/filtermfc9330cdw brother_lpdwrapper_mfc9330cdw ) || die

Looks like the name does not follow the same pattern as for the MFC-9320CDW but more like the MFC-9340CDW.
Which you can see here:
https://github.com/stefan-langenmaier/brother-overlay/blob/master/net-print/brother-mfc9340cdw-bin/brother-mfc9340cdw-bin-1.1.2.ebuild